l10n/en/privacy/cookie-settings.ftl

```diff
@@ -26,7 +26,7 @@ cookie-settings-turn-on-javascript = Please turn on JavaScript to update your Co
 cookie-settings-category-necessary-heading = Necessary
 cookie-settings-category-necessary-what-are-subheading = What are Necessary Cookies?
 cookie-settings-category-necessary-what-are-desc = These technologies are required to support essential website features, such as logging into a secure area of a website or adding items to an online shopping basket, and cannot be turned off.
-cookie-settings-category-necessary-how-does-desc = { -brand-name-mozilla } uses Necessary Cookies only to provide essential website features, such as logging in using your { -brand-name-mozilla } account or purchasing a software subscription at checkout. Without these technologies, essential website features may not function.
+cookie-settings-category-necessary-how-does-desc-v2 = { -brand-name-mozilla } uses Necessary Cookies only to provide essential website features, such as logging in using your { -brand-name-mozilla } account or purchasing a software subscription at checkout. Without these technologies, essential website features may not function.
```

Comment:
It took me a while to figure out that this is only on a branch (demo1, cc @alexgibson).

Given this, I didn't keep around the previous string as obsolete.

Reply:
I kept the same version originally to confirm that the linter failed. Force-pushed with the brand replacement.

Reply:
Looks like there is an issue filed for this (#14538)